src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -20,7 +22,19 @@ public void actionPerformed(ActionEvent e) {
 
 		if(e.getActionCommand().equals("Figure Joe 1"))
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-		else
+		else if(e.getActionCommand().equals("Figure Joe 2"))
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		else if(e.getActionCommand().equals("rectangle"))
+		{
+			DriverCommand command = CommandFactory.getSquare(-100, 0, 200);
+			command.execute(driverManager.getCurrentDriver());
+		}
+		else if(e.getActionCommand().equals("triangle"))
+		{
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.